package.json

```diff
@@ -4,7 +4,10 @@
     "description": "We Count Data Monitor project visualising Ontario Covid test centre data as accessible map",
     "scripts": {
         "lint": "fluid-lint-all",
-        "build": "node build.js",
+        "build:scss": "sass src/css/scss/fluid-covid-map-viz.scss src/css/fluid-covid-map-viz.css",
+        "watch:scss": "sass --watch src/css/scss/fluid-covid-map-viz.scss src/css/fluid-covid-map-viz.css",
+        "build": "npm run build:scss; node build.js",
```

Comment:
This syntax, I think, is not supported on Windows - it should probably be npm run build:css && node build.js

Reply:
Addressed and thanks.